roles/ovirt-vm-infra/library/ovirt_vms.py

```diff
@@ -0,0 +1,1255 @@
+#!/usr/bin/python
```

Comment:
Did you verified, this file is compatible with module_utils/ovirt.py from Ansible 2.2?